00_introduction/22_architecture_nodes.txt

```diff
@@ -17,6 +17,10 @@ GNU/Linux:
 
 * Ubuntu 10.04 LTS (Lucid), 12.04 LTS (Precise), 12.10 (Quantal), 14.04 LTS (Trusty)
 
+Other Unix systems:
+
+* IBM AIX 5.3, 6.1 and 7.1
```

Comment:
This looks good for 3.0 Will you also add Solaris and FreeBSD on the master branch?